tests/data/ar/Customer.php

```diff
@@ -47,7 +47,7 @@ public function getOrdersPlain()
 
     public function getOrders()
     {
-        return $this->hasMany(Order::className(), ['customer_id' => 'id'])->orderBy('id');
+        return $this->hasMany(Order::className(), ['customer_id' => 'id'])->orderBy('[[id]]');
```

Comment:
Shouldn't it work without specifying quotes explicitly?

Reply:
```php
1) yiiunit\framework\db\oci\ActiveRecordTest::testJoinWithAlias with data set #0 ('explicit')
yii\db\Exception: SQLSTATE[HY000]: General error: 918 OCIStmtExecute: ORA-00918: column ambiguously defined
 (ext\pdo_oci\oci_statement.c:157)
The SQL being executed was: SELECT "customer".* FROM "customer" INNER JOIN "order" "o" ON "customer"."id" = "o"."customer_id" WHERE ("o"."id"=1) AND ("customer"."id"=1) ORDER BY "id"
```

As there are several columns, the ordering column must be explicitly declared, otherwise it will generate an error, then we have two solutions or we remove the orderBy from the relationship, or I add a new relationship without the orderBy to use in the Oracle test.